src/assets/sass/protocol/base/elements/_details.scss

```diff
@@ -0,0 +1,46 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../includes/lib';
+
+.mzp-c-details {
+    .is-summary {
+        button {
+            all: inherit;
```

Comment:
Browser support is [sketchy](https://developer.mozilla.org/en-US/docs/Web/CSS/all#Browser_compatibility) here, is this usecase safe enough?

Reply:
Updated to declare specific properties to inherit.